src/Components/Patient/SampleDetails.tsx

```diff
@@ -431,7 +431,7 @@ export const SampleDetails = (props: SampleDetailsProps) => {
               {yesornoBadge(sampleDetails.patient_has_suspected_contact)}
             </div>
             {sampleDetails.patient_travel_history &&
-              sampleDetails.patient_travel_history !== "[]" && (
+              sampleDetails.patient_travel_history.length !== 0 && (
```

Comment:
@Pranshu1902 shouldn't it be 

```suggestion
              JSON.parse(sampleDetails.patient_travel_history).length !== 0 && (
```

Reply:
@khavinshankar I think it was correct as before. Getting this error after adding `JSON.parse`
![image](https://user-images.githubusercontent.com/70687348/215311025-4d96dac5-3390-47e6-92a7-cc0728a097d3.png)

Reply:
@Pranshu1902 cool, looks good